src/lang/en-US.ts

```diff
@@ -557,7 +560,7 @@ const Collections: ResolvedIntlConfig['messages'] = {
 
 const endpointConfigHeader = `Endpoint Config`;
 const EntityCreate: ResolvedIntlConfig['messages'] = {
-    'entityCreate.catalogEditor.heading': `Specification Editor`,
+    'entityCreate.catalogEditor.heading': `Advanced Specification Editor`,
```

Comment:
@oliviamiannone - we now hide the spec editor at the bottom of create/edit by default. Marking with Advanced now since only people that really know what they're doing should use it.

Reply:
That makes sense to me